pandas/tests/tools/test_to_datetime.py

```diff
@@ -717,6 +742,41 @@ def test_to_datetime_mixed_datetime_and_string_with_format(
             ),
         ],
     )
+    def test_to_datetime_mixed_offsets_with_none_tz(self, fmt, utc, expected):
+        # https://github.com/pandas-dev/pandas/issues/50071
+        msg = "parsing datetimes with mixed time zones will raise a warning"
+
+        with tm.assert_produces_warning(FutureWarning, match=msg):
+            result = to_datetime(
+                ["2000-01-01 09:00:00+01:00", "2000-01-02 02:00:00+02:00", None],
+                format=fmt,
+                utc=utc,
+            )
+        tm.assert_index_equal(result, expected)
+
+    @pytest.mark.parametrize(
+        "fmt, utc, expected",
+        [
+            pytest.param(
+                "%Y-%m-%d %H:%M:%S%z",
+                True,
+                DatetimeIndex(
+                    ["2000-01-01 08:00:00+00:00", "2000-01-02 00:00:00+00:00", "NaT"],
+                    dtype="datetime64[ns, UTC]",
+                ),
+                id="ISO8601, UTC",
+            ),
+            pytest.param(
+                "%Y-%d-%m %H:%M:%S%z",
+                True,
+                DatetimeIndex(
+                    ["2000-01-01 08:00:00+00:00", "2000-02-01 00:00:00+00:00", "NaT"],
+                    dtype="datetime64[ns, UTC]",
+                ),
+                id="non-ISO8601, UTC",
+            ),
+        ],
+    )
```

Comment:
is `utc` always equal to `True` in this test now? if so, can we remove it from the parametrisation and just pass `utc=True` inside the test?
there may some similar simplifications we can do now

Reply:
In this test `utc` is equal to `True` indeed. I removed `utc` from the parametrization.